README.md

```diff
@@ -56,7 +56,7 @@ cd pelux
 Instruct repo tool to fetch a manifest using the command `repo init`. In this context, branch denotes what
 branch of git repo `pelux-manifests` to use. Then make repo tool fetch all sources using the command `repo sync`.
 ```bash
-repo init -u https://github.com/Pelagicore/pelux-manifests.git -m <manifest> -b <branch>
+repo init -u https://github.com/Pelagicore/pelux-manifests.git -m manifests/<manifest> -b <branch>
```

Comment:
This change will need to be reflected in the SWF as well. Might be more than one place but at least there's this: https://pelagicore.github.io/software-factory/chapters/baseplatform/index.html#using-the-repo-tool